### PR TITLE
Update testing the blockchain application

### LINKED ISSUE #1527: Update `Testing the blockchain application`
### Description

* Update `Testing the blockchain application` to use v6 interfaces
* Update terminologies to Lisk SDK v6
* Update all examples using SDK v6

#### Affected branch / URL
`development`

#### Page location
`build-blockchain/test-app.adoc`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1527

### CODE REVIEW

**Review by Phanco — APPROVED** [review]
LGTM 👍

File: docs/modules/ROOT/pages/build-blockchain/test-app.adoc [hunk 1]
```diff
@@ -60,563 +64,389 @@ It is already possible to run the test at this point, though only the most basic
...
-        CHAIN_STATE_HELLO_COUNTER,
-        codec.encode(helloCounterSchema, { helloCounter: 0 })
-    );
-}
-----
+. The hello message is updated in the sender account with the specified hello string.
+. The hello counter is incremented by one.
 
-To verify that the function is implemented correctly, write 2 tests to check if the following occurs:
+Similar to the unit tests for the `verify()` function, a context is prepared using `createCommandExecuteContext()` for the `execute()` function which can be passed to the function when calling it in each test.
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/build-blockchain/test-app.adoc:340]: Just a note, maybe in the document somewhere you can mention that `testing` that is imported from SDK contains several helper functions to create contexts for verify/execute hooks, create a fake block header, etc that can be used to write tests.

**Review by ishantiw — APPROVED** [review]
LGTM, I minor comment to include some info

File: docs/modules/ROOT/pages/build-blockchain/test-app.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/build-blockchain/test-app.adoc:340]: Note added!

**Pull request merged**